test/system/components/student/editor_test.rb

```diff
@@ -95,7 +91,6 @@ class EditorTest < ApplicationSystemTestCase
         solution = create :concept_solution, user: user
 
         visit test_components_student_editor_path(solution_id: solution.id)
-        fill_in "Code", with: "Test"
```

Comment:
Why do all these get removed?

Reply:
Yes, that would also have been my question :)

Reply:
I didn't want to try and write JS for capybara to write into Monaco editor. Our tests right now work without this, so I'm leaving it out for now.